src/main.cpp

```diff
@@ -78,6 +64,12 @@ void opcontrol() {
       flywheelB.brake();
     }
 
```

Comment:
Overall, good work on this.  Make sure everything is tested and to the standard for the driver (i.e. adjusting deadbands, adjusting curves, etc.).  Here's an idea that could be worked on if there's time: make multiple drive settings dependent on the person driving (if we have more than 1 primary driver, of course).